cmd/provisioner/hostpath-provisioner.go

```diff
@@ -187,7 +197,7 @@ func (p *hostPathProvisioner) Delete(volume *v1.PersistentVolume) error {
 	if ann != p.identity {
 		return &controller.IgnoredError{Reason: "identity annotation on PV does not match ours"}
 	}
-	if !isCorrectNode(volume.Annotations, p.nodeName) {
+	if !isCorrectNode(volume.Annotations, p.nodeName, "kubevirt.io/provisionOnNode") {
```

Comment:
This should also check for the other annotation, we should delete on either one.

Reply:
When creating the PV, the provsioner puts the annotation on the PV:
https://github.com/kubevirt/hostpath-provisioner/blob/master/cmd/provisioner/hostpath-provisioner.go#L142
I think that the PV do not have the other annotation set.

Reply:
Good point, and I verified, regardless of annotation on PVC, it puts same annotation on PV.